### PR TITLE
Update the offboarding procedure

### PR DESCRIPTION
**Pull request by MarcinHoppe**
We offboarded some members of this WG after membership reviews (#591 and #632). The offboarding procedure was a bit out of date as not all referenced entities (GotHub teams, user lists and files) exist.

This PR updates the offboarding procedure so that it can be accurately followed the next time we need to offboard a WG member.

### CODE REVIEW

**Comment by MarcinHoppe** [comment]
@lirantal @vdeturckheim can you take a quick look? If it looks OK to you, I will merge.

File: processes/wg_offboarding.md [hunk 1]
```diff
@@ -16,16 +16,15 @@ At any time until the last time mark any member can chime in and request to reta
 ## Revoking Access to Confidential Systems
 
 The following is a check-list of actions to be taken upon departure of users from the Security WG (either voluntarily or due to inactivity as described above):
-* [ ] Remove user from [Repo README](https://github.com/nodejs/security-wg/blob/master/README.md), [Triage Team](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md), [Security Team List](https://github.com/nodejs/security-wg/blob/master/processes/security_team_members.md), [GitHub CodeOwners](https://github.com/nodejs/security-wg/blob/master/.github/CODEOWNERS), 
-* [ ] Remove membership from [Node.js WG GitHub Team](https://github.com/orgs/nodejs/teams/security-wg) and [Triage GitHub Team](https://github.com/orgs/nodejs/teams/ecosystem-security)
+* [ ] Remove user from [Repo README](https://github.com/nodejs/security-wg/blob/master/README.md), [Triage Team](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md)
+* [ ] Remove membership from [Node.js WG GitHub Team](https://github.com/orgs/nodejs/teams/security-wg)
```
Comment (lirantal) [inline processes/wg_offboarding.md:20]: why do we not need any longer to remove them from ecosystem-security?

Comment (MarcinHoppe) [reply processes/wg_offboarding.md:20]: @lirantal Because it does not exist. 🤣

Comment (lirantal) [reply processes/wg_offboarding.md:20]: Oh I think maybe you're just missing an invite :)
We don't use it officially so I guess no need to have it there but it does exist: https://github.com/orgs/nodejs/teams/ecosystem-security

But yeah agree we can skip in the onboarding process.

**Comment by lirantal** [comment]
LGTM let's merge

File: processes/wg_offboarding.md (hunk 1 above)
Comment (MarcinHoppe) [reply processes/wg_offboarding.md:20]: I get a 404 error when I click that link. Permissions issue?

Comment (lirantal) [reply processes/wg_offboarding.md:20]: I suspect so.

Comment (MarcinHoppe) [reply processes/wg_offboarding.md:20]: Let me merge this then, and perhaps we can file a separate issue about what to do with this team? Seems like we are not using it anyways.

**Pull request merged**